src/main/java/com/github/sgov/server/service/WorkspacePublicationService.java

```diff
@@ -58,6 +66,11 @@ public WorkspacePublicationService(final GithubRepositoryService githubService,
      * @return GitHub PR URL
      */
     public URI publish(URI workspaceUri) {
+        if (featureConf.isDemo()) {
+            throw new FeatureDisabledException("Publishing is disabled in configuration (see '"
+                    + featureConf.getClass().getAnnotation(ConfigurationProperties.class).value()
```

Comment:
Not very fancy IMO - the user of SGoV will never be interested in configuring the Feature him/herself. Would keep the message simple and focused to the enduser (to be shown in the UI possibly).

Reply:
Ok, I will fix it. 

I implemented it as I considered rather important consumers of SGoV REST API, not SGoV UI users.  But understand that you want to reuse it for UI messages  ...

Reply:
Imlemented.